framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1224,12 +1250,53 @@ private final AnnotatedTypeMirror fromMember(Tree tree) {
             return fromMemberTreeCache.get(tree).deepCopy();
         }
         AnnotatedTypeMirror result = TypeFromTree.fromMember(this, tree);
+
+        if (checker.hasOption("mergeStubsWithSource")) {
+            result = mergeStubsWithSource(result, tree);
+        }
+
         if (shouldCache) {
             fromMemberTreeCache.put(tree, result.deepCopy());
         }
+
         return result;
     }
 
+    /**
+     * Finds types from stubs and applies them to {@code type}.
```

Comment:
I'm also not sure what "Finds" means.  What kind of searching is done?
The documentation implies that the type is side-effected.  I think that probably isn't true.  Could you please clarify the documentation?

Reply:
The type is side-effected by `AnnotatedTypeCombiner`. I have clarified the documentation.